src/SAPIStatsTriggerHandler.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Drupal\sapi;
+
+use Drupal\Core\Config\ConfigManager;
+use Drupal\Core\Session\AccountProxy;
+use Drupal\Core\Routing\CurrentRouteMatch;
+
+/**
+ * Class SAPIStatsTriggerHandler.
+ *
+ * @package Drupal\sapi
+ */
+class SAPIStatsTriggerHandler implements SAPIStatsTriggerHandlerInterface {
+
+  /**
+   * Drupal\Core\Config\ConfigManager definition.
+   *
+   * @var Drupal\Core\Config\ConfigManager
+   */
+  protected $config_manager;
```

Comment:
As per Drupal coding standards [methods and class properties should use lowerCamel naming](https://www.drupal.org/node/608152#naming).